### PR TITLE
Add pytng

### PR DESCRIPTION
**Pull request by RMeli**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pytng/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    homepage = "https://pypi.org/project/pytng/"
+    pypi = "pytng/pytng-0.3.0.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.3.0", sha256="f563f9ea260ca8c8e17b3bcf9458bae35aedd5c58e1c5ac4dfff77a1e036506e")
+
+    depends_on("python@3.8:")
+
+    depends_on("py-cython@0.28:0.29.33", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pytng/package.py]:
```suggestion
    depends_on("py-cython@0.28:2", type="build")
```

File: var/spack/repos/builtin/packages/py-pytng/package.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    """
+
+    homepage = "https://pypi.org/project/pytng/"
+    pypi = "pytng/pytng-0.3.0.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.3.0", sha256="f563f9ea260ca8c8e17b3bcf9458bae35aedd5c58e1c5ac4dfff77a1e036506e")
+
+    depends_on("python@3.8:")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pytng/package.py]:
```suggestion
    depends_on("python@3.8:", type=("build", "run"))
```

File: var/spack/repos/builtin/packages/py-pytng/package.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    depends_on("py-cython@0.28:0.29.33", type="build")
+    depends_on("py-setuptools@42:", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pytng/package.py]: Where do you see this version requirement?

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-pytng/package.py (hunk 3 above)
Comment (RMeli) [reply var/spack/repos/builtin/packages/py-pytng/package.py]:
```suggestion
    depends_on("py-setuptools", type="build")
```

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**